src/test/resources/overbroadConstraintsTest/suggests_star_if_possible

```diff
@@ -0,0 +1,9 @@
+versions.props:
+com.example.*:*= 1.0.0
+
+versions.lock:
+com.example.moduleA:artifact-new:1.0.0 (2 constraints: abcdef1)
+com.example.moduleB:artifact-core:3.0.0 (2 constraints: abcdef1)
+
+new pins (blank when no pins needed):
+com.example.moduleB:* = 3.0.0
```

Comment:
The question here is why did we choose moduleB to add a new pin for rather than moduleA? If I were fixing this manually, I'd probably replace the pin with:

```
com.example.moduleA:* = 1.0.0
com.example.moduleB:* = 3.0.0
```

Reply:
so this comes from moduleA having the same version as the existing pin - currently I am only adding pins and never modifying existing ones. In most examples we could (and maybe should) replace the over-broad pin with a more constrained one

Reply:
I've pushed the changes needed to tighten the lose constraints